### DISCUSSION TITLE
Importing external script without installing Scripts Plugin via Webpack

### DISCUSSION THREAD

**Discussion by kuwts** [swup/swup#621] [Help] [answered]
Hey guys, 

Have an external script for a buy button from Ecwid, that looks like 

`<div class="ms-0 text-start ecsp ecsp-SingleProduct-v2 ecsp-Product ec-Product-45234524" itemtype="http://schema.org/Product" data-single-product-id="45234524"><div class="ecsp-title" itemprop="name" style="display:none;" content="White Door"></div><div customprop="options"></div><div customprop="qty"></div><div customprop="addtobag"></div></div><script data-cfasync="false" type="text/javascript" src="https://app.ecwid.com/script.js?12341234&data_platform=singleproduct_v2" charset="utf-8"></script><script type="text/javascript">xProduct()</script>`

I have a few different products that I set the button up for this way, is there any way I can have this script run on each product page? I really do not want to install webpack just for the Scripts Plugin, if there is a compiled version somewhere or if it is on some CDN I would love to hear about that option. Or if anyone has any other ideas would love to hear them as well. Thanks :)

**Comment by daun** [comment, marked as answer]
If it's on npm, it's on a CDN — try [unpckg](https://unpkg.com/browse/@swup/scripts-plugin@1.1.0/):

`https://unpkg.com/@swup/scripts-plugin@1.1.0/dist/SwupScriptsPlugin.min.js`

**Reply by kuwts** [reply]
You're the real MVP
